.stylelintrc.js

```diff
@@ -1,30 +1,9 @@
 /** @type {import('stylelint').Config} */
 module.exports = {
-  extends: ['@shopify/stylelint-plugin/prettier', './stylelint-polaris'],
+  extends: ['./stylelint-polaris'],
```

Comment:
modified this file so I could check against the migrator which as the config of only extending `stylelint-polaris`